examples/colors_rgb.rs

```diff
@@ -0,0 +1,159 @@
+/// This example shows the full range of RGB colors that can be displayed in the terminal.
+///
+/// Requires a terminal that supports 24-bit color (true color) and unicode.
+use std::{
+    error::Error,
+    io::{stdout, Stdout},
+    rc::Rc,
+    time::Duration,
+};
+
+use crossterm::{
+    event::{self, Event, KeyCode, KeyEventKind},
+    terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
+    ExecutableCommand,
+};
+use itertools::Itertools;
+use ratatui::{prelude::*, widgets::*};
+
+type Result<T> = std::result::Result<T, Box<dyn Error>>;
+
+fn main() -> Result<()> {
+    better_panic::install();
```

Comment:
Wow, nice crate! Should we also use this for the other examples for backtracing errors etc.?

Reply:
I think `better_panic` (or `human_panic`) should be part of all our examples + templates etc. It is a much nice developer experience and user experience with these installed.

I do have [a section in the book](https://ratatui-org.github.io/ratatui-book/how-to/setup-panic-hooks-better-panic.html) that describes how users can use `better_panic` so that's a start.

Reply:
Yep definitely - I'm searching for a consistent approach to use for all the examples instead of something different in each one. I think this is pretty close to the right balance of simplicity and functionality for examples. It's just enough to not have to pass around too much state. 

I wonder though whether the `init_terminal()`, `restore_terminal()` and `install_panic_hook()` methods might be worth putting at the bottom of the code outside of `impl App` instead.